robotiq_ft_sensor/package.xml

```diff
@@ -2,7 +2,7 @@
 <package format="2">
   <name>robotiq_ft_sensor</name>
   <version>1.0.0</version>
-  <description>Package for reading data from a Robotiq Force Torque Sensor</description>
+  <description><p2>Package for reading data from a Robotiq Force Torque Sensor</p2></description>
```

Comment:
Should this be a `<p>` instead of a `<p2>`?

Also: did `catkin_lint` really complain about this?

Reply:
yes and no, catkin_lint gives a warning if the description of the package starts with "package for" or "package to" or "this package..." saying that this is the boilerplate informaion. I am not familiar with these packages and I don't feel comfortable changing their description, then I decided to add there a new format to makes catkin_lint happy..

Reply:
again, thanks! I will update this